packages/infrastructure/authentik_atlas_mongodb_sso/README.md

```diff
@@ -0,0 +1,84 @@
+# Atlas MongoDB SSO with Authentik
+
+This module sets up the Authentik Application and Provider for Atlas MongoDB.
+
+## Sample Terragrunt Configuration
+
+```hcl
```

Comment:
We should set up a reference MongoDB Atlas instance for Panfactum and have the code for this in our reference environment. I dislike having code examples like this that are not "live" if at all possible to avoid as they will quickly get out of date.